corehq/form_processor/tests/test_sharding.py

```diff
@@ -19,7 +20,8 @@ class ShardingTests(TestCase):
 
     @classmethod
     def setUpClass(cls):
-        cls.partion_config = PartitionConfig()
+        with trap_extra_setup(PartitionedDatabaseRequiredError):
```

Comment:
heh, I didn't even notice the `skipUnless` decorator. I guess it's not working with nose. I'll check into it.